components/web/goaccess/manifests/sample-manifest.p5m

```diff
@@ -22,14 +22,16 @@ set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
 
 license $(COMPONENT_LICENSE_FILE) license='$(COMPONENT_LICENSE)'
 
-file path=etc/goaccess.conf
+file path=etc/goaccess.conf preserve=true
+file path=etc/goaccess/browsers.list preserve=true
+file path=etc/goaccess/goaccess.conf preserve=true
+file path=etc/goaccess/podcast.list
```

Comment:
Never touch the sample-manifest. Only generate it and add to your PR if it has changes other than the year.
With this you (or any other maintainer) can check whether something has changed since the last release.